packages/fd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+singularity run ~/.planutils/bin/images/downward.simg $@
```

Comment:
I think, they now use `.sif` as an extension for singularity files. Also, this would work as 
`~/.planutils/bin/images/downward.simg $@` as well (no need for `singularity run`).

Reply:
For IPC planners, we could do something more fancy here that would actually require `singularity run`, like creating a temporary working directory and copying the input files there.

Reply:
Not sure I follow the IPC comment...would the cmd-line change?

Reply:
The command line would stay the same, but the `run` script would internally first create a temporary directory, then copy the input files there, then run singularity with the temporary directory mounted as its home and current directory. After the singularity process completes (terminates or crashes) the temporary directory is deleted. This covers planners that have additional requirements such as that the input has to be in the working directory. Not sure if this should be done for all IPC planners or only ones that have such a requirement.

Reply:
Ah, right. And also covers temporary files that are used (like output.sas used to be). I think this would be a good additional example to have in the `TEMPLATE/run` file. Going to try and push through the vanilla version first though.